src/main/resources/static/api.yml

```diff
@@ -5,6 +5,171 @@ info:
 servers:
 - url: /
 paths:
+  /cas2/applications:
+    post:
+      tags:
+        - Operations on CAS2 applications
+      summary: Creates a CAS2 application
+      parameters:
+        - name: createWithRisks
```

Comment:
can we get rid of this `createWithRisks` parameter?

Reply:
yes, well spotted -- this has just been removed from CAS2's application entity in https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/968 -- and was not being passed to the `ApplicationService` in any case...